### ISSUE TITLE
[Bug]: incorrect / 'area repeateated' thumbnails generated

### ISSUE THREAD

**Issue by gsxgmpkt9020** [ollm/OpenComic#302]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/ollm/OpenComic/issues) for a bug report that matches the one I want to file, without success.

### OpenComic Version

1.3.1

### Operating System

WIN 10 22H2 19045.3324

### Installed via

.exe

### Steps to reproduce

export cbz files, open the library

### Expected Behavior

correct thumnails for all files

### Actual Behavior

 incorrect / 'area repeateated' thumbnails  for some files

![20241211135321001](https://github.com/user-attachments/assets/62cf561f-7e57-405e-a642-5aae48f4f204)
![20241211135358002](https://github.com/user-attachments/assets/a937c732-dbba-48cf-a0a6-72633ca783d5)
![20241211135441003](https://github.com/user-attachments/assets/73f3e3f3-f0fc-4a5e-8434-db8a31c7dbd7)
![20241211135600004](https://github.com/user-attachments/assets/f19a0325-1d07-4be8-a5f4-18a737468a97)

### Error message

None

### Additional Information

I remember this will not happen on 1.2.0 version app

**Comment by ollm** [comment]
I have made some changes to fix this bug c415b3f0f6eb4bd6eb1bdd6cdd8a191b809df91e

This happens because `7-zip` or `node-7z` sometimes gives as extracted an file/image while it is still writing it to disk, so OpenComic starts generating the thumbnail with the partially extracted image. Previously in `v1.2.0`, in this cases the thumbnails were not generated, since the library used for images (`sharp`) provides an error if the image is corrupted, but in `v1.3.0` I enabled in `sharp` an option to force work in corrupted images to fix this, I did some testing and it seemed to generate thumbnails correctly, but it only seems to do so for progressive JPGs and not regular PNGs or JPGs.

With the new fix, the behavior is now the following.

- The thumbnail is generated with the work in corrupted images option disabled.
- If it gives an error, it is tried again after a 100 milliseconds 2 more times.
- If the thumbnail still cannot be generated, it tries once more time but with the option to work in corrupted images enabled.

Buid: https://mega.nz/file/SPIUkATJ#f5zq2Ku3N-FoJBWc0yqVCcBrazBwEbiJSn5A5rEqSws
